lib/potentialbuild.rb

```diff
@@ -1114,7 +1114,7 @@ def post_results compiler, pending
           $logger.info("Posting pending results file");
           response =  github_query(@client) { @client.create_contents(@config.results_repository,
                                              "#{@config.results_path}/#{get_branch_folder()}/#{@dateprefix}-#{results_file_name compiler}",
-                                             "Commit initial build results file: #{@dateprefix}-#{results_file_name compiler}", 
+                                             "Commit initial build results file: #{@dateprefix}-#{Socket.gethostname}-#{results_file_name compiler}", 
```

Comment:
Ah, I got it.  I never paid attention to the underlying file names.  I'll move it to the front of that comment.